src/witan/models/dem/ccm/fert/hist_asfr_age.clj

```diff
@@ -11,15 +11,15 @@
 (defn ->births-data-year
   "Takes births-data dataset
    Returns maximum value in Year column of births-data"
-  [{:keys [births-data]}]
+  [{:keys [births-data]} _]
```

Comment:
you can use it if you want to pass the function an argument but you don't use that argument at all. In these cases, no parameters are used- but from the way the workflow is set up, the parameters still need to be passed to every function.